docs/en/1_exist_data_model.md

```diff
@@ -510,6 +507,22 @@ Training requires preparing datasets too. See section [Prepare datasets](#prepar
 Currently, the config files under `configs/cityscapes` use COCO pretrained weights to initialize.
 You could download the existing models in advance if the network connection is unavailable or slow. Otherwise, it would cause errors at the beginning of training.
 
+### Learning rate automatically scale
+
+If you want to enable this feature, you need to add argument `--auto-scale-lr`.
+The basic usage is as follows.
+
+```shell
+python tools/train.py \
+    ${CONFIG_FILE} \
+    --auto-scale-lr \
+    [optional arguments]
+```
+
+If you enabled this feature, the learning rate will be automatically scaled according to the number of GPUs of the machine and the batch size of training. See [linear scaling rule](https://arxiv.org/abs/1706.02677) for details. For example, If there are 4 GPUs and 2 pictures on each GPU, 'lr = 0.01', then if there are 16 GPUs and 4 pictures on each GPU, it will automatically scale to 'lr = 0.08'. 
+
+If you don't want to use it, the default learning rate in config files is based on batch size of 16, you need to calculate the learning rate manually according to the practical training batch size and change the learning rate in specific config file. 
```

Comment:
We need to mention the function of `default_batch_size` and `enable_auto_scale_lr` in config.

Reply:
Done